### ISSUE TITLE
A11y: Create high level accessibility checklists for UX teams

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#115730]
## Issue
In the interest of accessibility education and keeping simpler accessibility issues from making it too far down the development pipeline, I was thinking of making an accessibility checklist for UX to use against their designs.

I don't want the checklist to be an additional bit of work for UX to do, but instead to be a resource they can use to gut-check their design.

The checklist shouldn't be exhaustive, but should cover the main things that I look for when doing a11y review of wireframes

## Acceptance Criteria
- [x] Create a checklist of accessibility issues to check for in a Figma wireframe
- [x] Review checklist against VA's accessibility requirements / update
- [x] Share with the team
- [x] Save checklist in the appropriate place where I can return and easily make edits - GitHub? Slack Canvas? Sharepoint

## Resources
- [ADE Figma prototype accessibility review tempalte](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/ADE/templates/design-review-template.md)

**Comment by bacitracin** [comment]
Created and shared in Slack
Saved in Github [BMT](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/claim-appeal-status/accessibility/ux-accessibility-checklist.md) / [Decision Reviews](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/accessibility/ux-accessibility-checklist.md)
and as a [Slack Canvas](https://dsva.slack.com/docs/T03FECE8V/F09B1SN8HT6)
